Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -93,4 +126,44 @@ namespace AzToolsFramework::Prefab
         return instance.has_value() && (&instance->get() == &m_focusedInstance->get());
     }
 
+    const AZ::IO::Path& PrefabFocusHandler::GetPrefabFocusPath()
```

Comment:
question: Could this be `const`? (repeated for other calls below) - even if this is an EBus they can be `const` and likely should be as we start using interfaces more